CriticalMass/AppController.swift

```diff
@@ -46,17 +48,32 @@ class AppController {
     }
 
     private func getSettingsViewController() -> SettingsViewController {
-        return SettingsViewController()
+        return SettingsViewController(themeController: themeController)
     }
 
     lazy var rootViewController: UIViewController = {
         let rootViewController = MapViewController()
 
+        let kniggeButton: UIButton = {
+            let button = UIButton(type: .system)
+            button.setImage(UIImage(named: "Knigge"), for: .normal)
+            return button
+        }()
+        let settingsButton: UIButton = {
+            let button = UIButton(type: .system)
+            button.setImage(UIImage(named: "Settings"), for: .normal)
+            return button
+        }()
+
         let navigationOverlay = NavigationOverlayViewController(navigationItems: [
-            .init(representation: .view(rootViewController.followMeButton), action: .none),
-            .init(representation: .button(chatNavigationButtonController.button), action: .navigation(viewController: getSocialViewController)),
-            .init(representation: .icon(UIImage(named: "Knigge")!, accessibilityLabel: NSLocalizedString("rules.title", comment: "")), action: .navigation(viewController: getRulesViewController)),
-            .init(representation: .icon(UIImage(named: "Settings")!, accessibilityLabel: NSLocalizedString("settings.title", comment: "")), action: .navigation(viewController: getSettingsViewController)),
+            .init(representation: NavigationOverlayItem.Representation(button: rootViewController.followMeButton),
+                  action: .none),
+            .init(representation: NavigationOverlayItem.Representation(button: chatNavigationButtonController.button),
+                  action: .navigation(viewController: getSocialViewController)),
+            .init(representation: NavigationOverlayItem.Representation(button: kniggeButton),
+                  action: .navigation(viewController: getRulesViewController)),
+            .init(representation: NavigationOverlayItem.Representation(button: settingsButton),
+                  action: .navigation(viewController: getSettingsViewController)),
         ])
```

Comment:
Updated these to be buttons also for accessability

Reply:
Makes applying a theme easier since it is only one kind. For using the app with voice over it also makes sense. Plus shortens code

Reply:
It doesn't make a difference for VoiceOver and Theming because both solution are using UIButtons in the end. I don't like that we know have UIButton instances in the AppController

Reply:
I updated to PR to roll back to the previous solution